packages/databricks-vscode/src/cli/CliWrapper.ts

```diff
@@ -342,57 +417,23 @@ export class CliWrapper {
         configfilePath?: string,
         logger?: logging.NamedLogger
     ) {
-        const cmd = [
+        return await runBundleCommand(
+            "summarize",
             this.cliPath,
-            "bundle",
-            "summary",
-            "--target",
-            target,
-            // Forces the CLI to regenerate local terraform state and pull the remote state.
-            // Regenerating terraform state is useful when we want to ensure that the provider version
-            // used in the local state matches the bundled version we supply with the extension.
-            "--force-pull",
-        ];
-
-        logger?.info(
-            `Refreshing bundle configuration for target ${target}...`,
-            {bundleOpName: "summarize"}
+            ["bundle", "summary", "--target", target],
```

Comment:
`--foce-pull` is still needed though, otherwise we will have problems during updates that bring new CLI with new databricks provider version.